dev-support/Jenkinsfile

```diff
@@ -628,6 +630,122 @@ pipeline {
             }
           }
         }
+
+        stage ('yetus jdk17 hadoop3 checks') {
+          when {
+              branch 'master'
```

Comment:
Is this the correct syntax? IIRC at least we need to specify anyOf or allOf?

Reply:
@Apache9 followed this which is expected syntax.

https://www.jenkins.io/doc/book/pipeline/syntax/#built-in-conditions

Reply:
I think would be better to make it.  Will update it.
`when { anyOf { branch 'branch-3'; branch 'master' } }`